src/main/java/gov/usgs/cida/pubs/domain/ContributorType.java

```diff
@@ -17,6 +17,7 @@
 	public static final Integer COMPILERS = 3;
 	public static String AUTHOR_KEY = "authors";
 	public static String EDITOR_KEY = "editors";
+	public static String COMPILER_KEY = "compilers";
```

Comment:
A block should be added in onApplicationEvent to get this value from the database.
Not the biggest fan of this pattern, but the text can be changed in the database and we need to actual value from there to tie everything together. These values up here are more just a default convenience for testing...